scss/boost_union/post.scss

```diff
@@ -112,6 +112,20 @@
     .login a:hover {
         color: $white;
     }
+
```

Comment:
Try to prepend this by

/*csslint important:false*/

I think you give the rationale to use this in the comment. The above line would then shut csslint quiet.
Here's a usage example: https://github.com/moodle/moodle/blob/MOODLE_402_STABLE/lib/yuilib/3.18.1/cssnormalize/cssnormalize.css#L565

Thanks for your contribution, @wa-bea !

Best,
Luca

Reply:
Hi Luca,

thank you for this hint how to fix the linter warning!

However, may I propose to use a slightly different syntax which we already use in Boost Union?
If you look at https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/blob/79e568dd9c5a3c81440edd836c51755f2c9b2344/scss/boost_union/post.scss#L77, you will see how it's done and Beata can directly adopt it to her code.

Reply:
Ah, yes, the inline one!
That's much more elegant, correct.